File: 백준/Bronze/2576. 홀수/홀수.swift
import Foundation

var nums:[Int] = []
for _ in 0..<7 { nums.append(Int(readLine()!)!) }

var min_odd = 1000
var odd_sum = 0
for n in nums {
    if (n%2 == 1) {
        odd_sum += n
        min_odd = (min_odd > n) ? n : min_odd
    }
}
if (odd_sum > 0) {
    print(odd_sum)
    print(min_odd)
} else {
    print(-1)
}

File: 백준/Bronze/2576. 홀수/홀수.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int odd_sum = 0;
        int min_odd = 1000;
        for (int i=0; i<7; i++) {
            int n = Integer.parseInt(br.readLine());
            if (n%2 == 1) {
                odd_sum += n;
                if (min_odd > n) min_odd = n;
            }
        }

        if (odd_sum > 0) {
            System.out.println(odd_sum);
            System.out.println(min_odd);
        } else {
            System.out.println(-1);
        }
    }
}